### ISSUE TITLE
Facing issues while using it in colab

### ISSUE THREAD

**Issue by KShivendu** [eti-p-doray/unet-gan-matting#27]
After some experimentation, I found that chromedriver doesn't work in colab. I used `!./chromedriver` in colab cell

The result was 
`src/tcmalloc.cc:283] Attempt to free invalid pointer 0x1951c81affc0`

But when i try locally `./chromedriver` :  

`
Starting ChromeDriver 78.0.3904.105 (60e2d8774a8151efa6a00b1f358371b1e
0e07ee2-refs/branch-heads/3904@{#877}) on port 9515
Only local connections are allowed.
Please protect ports used by ChromeDriver and related test frameworks
to prevent access by malicious code.

Any suggestions what can be done ?? 
I have tried
`
!sudo apt-get install libjemalloc-dev 
!apt install libtcmalloc-minimal4
`
`

**Comment by adnangif** [comment]
Hey, Did you find any solution to this problem?
I'm also facing this issue.

**Comment by KShivendu** [comment]
No. But you can try [U-2 Net](https://github.com/NathanUA/U-2-Net) for background removal. It seems to be the SOTA.
